plugins/modules/dedicated_server_ola_unconfigure_wait.py

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+from ansible.module_utils.basic import AnsibleModule
+
+DOCUMENTATION = '''
+---
+module: dedicated_server_ola_unconfigure_wait
```

Comment:
isn't this the exact same code as `dedicated_server_ola_configure_wait` ? can''t this be factorised ?